packages/examples/src/Feedback/FeedbackContainer.stories.ts

```diff
@@ -0,0 +1,104 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
+import { FeedbackContainer } from "./FeedbackContainer";
+import type { NovaEnvironmentDecoratorParameters } from "@nova/react-test-utils";
+import {
+  getEnvForStory,
+  getNovaEnvironmentDecorator,
+  MockPayloadGenerator,
+} from "@nova/react-test-utils";
+import { getSchema } from "../testing-utils/getSchema";
+import type { TypeMap } from "../__generated__/schema.all.interface";
+
+const schema = getSchema();
+
+const meta: Meta<typeof FeedbackContainer> = {
+  component: FeedbackContainer,
+  decorators: [getNovaEnvironmentDecorator(schema)],
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  parameters: {
+    novaEnvironment: {
+      resolvers: {
+        Feedback: () => sampleFeedback,
+      },
+    },
+  } satisfies NovaEnvironmentDecoratorParameters<TypeMap>,
```

Comment:
Would be nice if we can get rid of this in the future.

Reply:
of `satisfies`? Then probably no, we won't get rid of it. We still need a way to strongly type `parameters` (by default it's just generic `Record`) so we either need to use satisfies or adjust story type itself

Reply:
![image](https://github.com/microsoft/nova-facade/assets/19153565/b2c8bba1-b3c4-4a8f-bad4-8dc36e99e2c5)

Reply:
Yeah the satisfies part. I figured this was the reason, but I’m thinking that it should be very possible to explain users how they can create their own StoryObj type that composed the normal StoryObj and a resolvers map with their schema.

Reply:
(To be clear, this is not a blocker!)

Reply:
One can always do `const Primary: StoryObj & {parameters: NovaEnvironmentDecoratorParameters<TypeMap>}` but I personally like satisfies part :)